README.md

```diff
@@ -92,7 +92,7 @@ flannel plugin will set the following fields in the delegated plugin configurati
 
 Additionally, for the bridge plugin, `isGateway` will be set to `true`, if not present.
 
-One use case of the `ipam` configuration is to allow adding back the routes to the cluster services and/or to the hosts when using `isDefaultGateway=false`. In that case, the bridge plugin does not install a default route and, as a result, only pod-to-pod connectivity would be available.
+One use case of the `delegate` configuration is to allow adding back the routes to the cluster services and/or to the hosts when using `isDefaultGateway=false`. In that case, the bridge plugin does not install a default route and, as a result, only pod-to-pod connectivity would be available.
```

Comment:
Thanks for contributing. You have a point but I would rephrase it because `delegate` configuration does not make much sense. What about this:
```
Additionally, for the bridge plugin, `isGateway` will be set to `true`, if not present. The bridge plugin includes ``isDefaultGateway` that when false allow adding back the routes to the cluster services and/or to the hosts. In that case, the bridge plugin does not install a default route and, as a result, only pod-to-pod connectivity would be available.
```